src/SFA.DAS.Roatp.CourseManagement.Application/ProviderLocations/Commands/DeleteProviderLocation/DeleteProviderLocationCommandHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Roatp.CourseManagement.Domain.Interfaces;
+
+namespace SFA.DAS.Roatp.CourseManagement.Application.ProviderLocations.Commands.DeleteProviderLocation;
+
+public class DeleteProviderLocationCommandHandler : IRequestHandler<DeleteProviderLocationCommand, Unit>
```

Comment:
You do not have to have return type any more, so you just remove the Unit

Reply:
Well did it and it worked, but I was expecting it to break things :)